packages/cli/internal/pkg/cli/workflow/manager.go

```diff
@@ -339,7 +335,12 @@ func (m *Manager) uploadInputsToS3() {
 		return
 	}
 	objectKey := awsresources.RenderBucketDataKey(m.projectSpec.Name, m.userId)
-	updateInputs, err := m.InputClient.UpdateInputs(m.Project.GetLocation(), m.input, m.bucketName, objectKey)
+	absInputsPath, err := filepath.Abs(m.inputsPath)
+	if err != nil {
+		m.err = err
```

Comment:
We need to return here if we encounter an error

Reply:
Good catch. Thank you!